data/src/main/java/io/gnosis/data/repositories/SafeRepository.kt

```diff
@@ -93,6 +93,8 @@ class SafeRepository(
 
         val DEFAULT_FALLBACK_HANDLER = BuildConfig.DEFAULT_FALLBACK_HANDLER.asEthereumAddress()!!
         const val DEFAULT_FALLBACK_HANDLER_DISPLAY_STRING = "DefaultFallbackHandler"
+        const val DEFAULT_FALLBACK_HANDLER_UNKNOWN_DISPLAY_STRING = "Unknown"
+
         const val SAFE_MASTER_COPY_UNKNOWN_DISPLAY_STRING = "Unknown"
```

Comment:
Why are we hardcoding display strings?

Reply:
that is the reason I called it workaround. Here is quick fix that will allow users to see correct information. Harcoded strings will be dealt with in https://app.zenhub.com/workspaces/safe-multisig---mobile-5c1b98250e13551e8aae81eb/issues/gnosis/safe-android/964